composer.json

```
@@ -102,7 +102,7 @@
         "scheb/2fa-bundle": "^6.0 || ^7.0",
         "scheb/2fa-trusted-device": "^6.0 || ^7.0",
         "scrivo/highlight.php": "^9.18",
-        "scssphp/scssphp": "^1.5",
+        "scssphp/scssphp": "^2.0",
```

```suggestion
        "scssphp/scssphp": "^1.5 || ^2.0",
```
Since there are no code changes, I think we should just allow both versions. This will allow the Oveleon package continue to be used in Contao 5.5+ without any changes for example.

---

I agree, however I'm pretty certain that there's changes from libsass / non-spec features in scssphp 1.x, that may not work in 2.x, but that would just be invalid CSS 😃